Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -92,6 +96,7 @@
                 $scope.contractTemplates = contractTemplates;
                 $scope.purchaseForms = purchaseForms;
                 $scope.procurementStrategies = procurementStrategies;
+                $scope.criticalities = criticalities;
```

Comment:
look in DPR how choice types are mapped to view models and updates are handled with service calls